repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/legacy-publication.ftl

```diff
@@ -134,7 +134,8 @@
                 <div class="panel panel--grey push-half--bottom">
                     <h3><@fmt.message key="headers.resources"/></h3>
                     <ul data-uipath="ps.publication.resources">
-                        <#list legacyPublication.attachments as attachment>
+                        ${legacyPublication.getExtAttachments()?size}
```

Comment:
Do we need that in the template?

Reply:
@robValtech could you have a look at this?